src/piqtree2/iqtree/_tree.py

```diff
@@ -184,6 +188,9 @@ def build_tree(
         The substitution model with base frequencies and rate heterogeneity.
     rand_seed : Optional[int], optional
         The random seed - 0 or None means no seed, by default None.
+    bootstrap_replicates : int, optional
+        The number of bootstrap replicates to perform, by default 0.
+        At least 1000 is recommended.
```

Comment:
I though this wasn't recommended, but instead required if bootstrapping was to be performed.